src/ucp/core/ucp_mm.c

```diff
@@ -1187,6 +1187,44 @@ ucs_status_t ucp_mem_unmap(ucp_context_h context, ucp_mem_h memh)
     return UCS_OK;
 }
 
+static ucs_status_t
```

Comment:
If we always use one md for mtype, maybe we can avoid ucp_memh as well, just use uct objects?